### ISSUE TITLE
mask rcnn + rle hyper parameters

### ISSUE THREAD

**Issue by ssssholmes** [Jeff-sjtu/res-loglikelihood-regression#12]
Hi, I want to train mask rcnn with rle loss！
1. Could you give me some details about the optimizer and learning rate scheduler used in your experiments? I found rle uses the adam while the original mask rcnn uses the sgd?
2. Do you use the exactly same flow model architecture as released in this codebase?
3. Does the rle loss directly regress absolute coordinates of the image or relative coordinates of the pred boxes? (e.g. normalize the target keypoint to range [-0.5, 0.5])
 Thanks!

**Comment by ssssholmes** [comment]
![屏幕快照 2021-08-29 15 48 04](https://user-images.githubusercontent.com/48430696/131242890-3237d697-1c10-46c3-96bf-317d7f9c5122.png)

I tried it and the perfermance at the beginning is pretty pool.

**Comment by jeffffffli** [comment]
Hi @ssssholmes,

1. We use the original optimizer (sgd) and learning rate in Mask R-CNN, with 180000 iters. And the. Mask R-CNN model is pre-trained with heatmap.
2. Yes, the flow model is the same.
3. We regress the relative coordinates.

**Comment by ssssholmes** [comment]
@Jeff-sjtu , Thanks for your reply! One more question, Do you use normalization in the linear layers for the output of sigma in mask rcnn?

**Comment by jeffffffli** [comment]
> @Jeff-sjtu , Thanks for your reply! One more question, Do you use normalization in the linear layers for the output of sigma in mask rcnn?

We don't use normalization in the linear layer for sigma. We just used the sigmoid function after the linear layer.

**Comment by WeianMao** [comment]
> Hi @ssssholmes,
> 
> 1. We use the original optimizer (sgd) and learning rate in Mask R-CNN, with 180000 iters. And the. Mask R-CNN model is pre-trained with heatmap.
> 2. Yes, the flow model is the same.
> 3. We regress the relative coordinates.

thanks for your great work. i prepare to site your paper. but i meat the mask rcnn's setting issue. in your paper, you said mask rcnn is trained without heatmap pretrain and the trainin schedule is 3X (27000iter). but this is different from the setting you just mentioned above. so can you please provide the excat setting of the mask rcnn you used in you paper? thanks a lot!

**Comment by jeffffffli** [comment]
Hi @maowayne123,

In Table. 2 in the paper, both the default Mask RCNN and Mask RCNN + RLE are trained with 270,000(3x) iterations. After submission, we retrain Mask RCNN + RLE with the 2x schedule (to see if we can save some time), we found the performance basically the same as the 3x schedule (still 66.7 mAP).

**Comment by WeianMao** [comment]
great! thank you!
